cc/private/toolchain/unix_cc_configure.bzl

```diff
@@ -196,8 +196,11 @@ def _find_linker_path(repository_ctx, cc, linker, is_clang):
 
     # Extract linker path from:
     # /usr/bin/clang ...
-    # "/usr/bin/ld.lld" -pie -z ...
-    linker_command = result.stderr.splitlines()[-1]
+    #  "/usr/bin/ld.lld" -pie -z ...
+    invocations = [line for line in result.stderr.splitlines() if line.startswith(" \"")]
```

Comment:
Could you add a comment to the code why (or at least that) a) the linker is know to be invoked with quotes around it and b) why or that it always has exactly one space at the beginning of the line.
I guess those 2 are the reason/motivation for this statement, aren't they?